cli/cli-client/src/main/java/org/infinispan/cli/commands/Put.java

```diff
@@ -19,34 +20,52 @@
  * @author Tristan Tarrant &lt;tristan@infinispan.org&gt;
  * @since 10.0
  **/
-@MetaInfServices(CliCommand.class)
-@CommandDefinition(name = "put", description = "Puts an entry into the cache", activator = ConnectionActivator.class)
+@MetaInfServices(Command.class)
+@CommandDefinition(name = Put.CMD, description = "Puts an entry into the cache", activator = ConnectionActivator.class)
 public class Put extends CliCommand {
+   public static final String CMD = "put";
+   public static final String ENCODING = "encoding";
+   public static final String TTL = "ttl";
+   public static final String MAX_IDLE = "max-idle";
+   public static final String IF_ABSENT = "if-absent";
+
    @Arguments(required = true)
    List<String> args;
 
-   @Option(completer = EncodingCompleter.class)
+   @Option(completer = EncodingCompleter.class, shortName = 'e')
    String encoding;
 
-   @Option(completer = CacheCompleter.class)
+   @Option(completer = CacheCompleter.class, shortName = 'c')
    String cache;
 
-   @Option(completer = FileOptionCompleter.class)
+   @Option(completer = FileOptionCompleter.class, shortName = 'f')
    Resource file;
 
+   @Option(shortName = 'l', defaultValue = "0")
+   long ttl;
+
+   @Option(name = "max-idle", shortName = 'i', defaultValue = "0")
```

Comment:
Should default be -1?

Reply:
No, default should be 0 (use server default)